lib/src/wolt_modal_sheet.dart

```diff
@@ -40,7 +40,7 @@ class WoltModalSheet<T> extends StatefulWidget {
   final ValueNotifier<int> pageIndexNotifier;
   final VoidCallback? onModalDismissedWithBarrierTap;
   final Widget Function(Widget)? decorator;
-  final WoltModalType Function(BuildContext context) modalTypeBuilder;
+  final WoltModalTypeBuilder modalTypeBuilder;
```

Comment:
Shouldn't this be also nullable as we assign the default within WoltModalSheetRoute?

Reply:
No because the Route provides the non-null `WoltModalTypeBuilder` for this widget. Do you think that we should make it nullable in the widget level and assign the default value inside the widget? I think it makes sense to provide non-null to the widget so that we don't worry about the null check inside the widget. What do you think @MuTe33 ?